src/models/orderdetails.py

```diff
@@ -0,0 +1,20 @@
+"""
+Define the OrderDetails model
+"""
+from . import db
+from .abc import BaseModel, MetaBaseModel
+
+
+class OrderDetails(db.Model, BaseModel, metaclass=MetaBaseModel):
+    """OrderDetails Model"""
+
+    __tablename__ = "order_details"
+
+    id = db.Column(db.Integer, primary_key=True)
+    status = db.Column(db.Boolean, nullable=False)
```

Comment:
Move to foreign key, a different model will be created for status

Reply:
Will the status still be **boolean** type?